pkg/apis/certmanager/v1alpha2/types_certificate.go

```diff
@@ -145,6 +150,21 @@ type CertificateSpec struct {
 	KeyEncoding KeyEncoding `json:"keyEncoding,omitempty"`
 }
 
+type X509DistinguishedName struct {
+	// Country/Region
+	Country            []string `json:"country,omitempty"`
+	Organization       []string `json:"organization,omitempty"`
```

Comment:
Please omit this field from this struct for now. As part of v1alpha3, we can remove the top-level `spec.organization` field and rename it to `spec.subject.organizations`, but we should not duplicate the field in v1alpha2 😄